### PR TITLE
create db instance in the correct region

### PR DESCRIPTION
**Pull request by lanej**
* and fix variable shadow warning

### CODE REVIEW

File: lib/fog/aws/requests/rds/describe_db_instances.rb [hunk 1]
```diff
@@ -46,7 +46,6 @@ def describe_db_instances(identifier=nil, opts={})
              case server["DBInstanceStatus"]
              when "creating"
                if Time.now - server['InstanceCreateTime'] >= Fog::Mock.delay * 2
-                 region = "us-east-1"
                  server["DBInstanceStatus"] = "available"
                  server["AvailabilityZone"] ||= region + 'a'
```
Comment (geemus) [inline lib/fog/aws/requests/rds/describe_db_instances.rb]: Won't this `region` be undefined now?

Comment (lanej) [inline lib/fog/aws/requests/rds/describe_db_instances.rb]: it's pulling from the services' region https://github.com/fog/fog-aws/blob/master/lib/fog/aws/rds.rb#L141

Comment (geemus) [inline lib/fog/aws/requests/rds/describe_db_instances.rb]: Ah, got it, just wanted to double check.

**Comment by geemus** [comment]
Thanks!

**Pull request merged**